givitsite/coordinate/models.py

```diff
@@ -1,3 +1,23 @@
 from django.db import models
+from friendreq.models import REGION_CHOICES 
+from friendreq.models import ITEM_CHOICES
 
-# Create your models here.
+
+STATUS_CHOICES = [
+    ('coordinated', 'coordinated'),
+    ('waiting', 'waiting')
+]
+
+
+class CoordinatedItems(models.Model):
+    #friennd ID inherite from user table
```

Comment:
It's not seems that it inherit from user table and I didnt find user table. Where can I find it?
Also, as far as I know each class would create a new table - are you sure that this is what you meant to do?

Reply:
Your right I removed the comment